script.js

```diff
@@ -24,4 +21,80 @@ function episodeCode(season, number) {
   let code = `S${s}E${num}`;
   return code;
 }
-window.onload = setup;
+
+const searchBox = document.getElementById("search");
+searchBox.addEventListener("input", handleSearchInput);
```

Comment:
Couple of bits of code adding event listeners, I would probaly put these together for tidiness